packages/react-components/icons/RadioButton.tsx

```diff
@@ -0,0 +1,50 @@
+import colors from '@celo/react-components/styles/colors'
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import Svg, { Circle } from 'svgs'
+
+
+interface Props {
+  height?: number
+  color?: string
+  selected?: boolean
+  disabled?:boolean
+}
+
+const styles = StyleSheet.create({
+  circle: {
+    paddingTop: 2,
+    height: 20,
+    width: 20,
+    borderRadius: 12,
+    borderWidth: 2,
+    borderColor: colors.gray4,
+  },
+})
+
+export default class RadioButton extends React.PureComponent<Props> {
```

Comment:
No need to change it now, but for the future, it's better to use functional components instead of classes. Mainly because it's the direction React is going and class components may not be supported anymore in future versions or become "second class citizens" like not receiving performance improvements or being actively maintained anymore.

Reply:
totally- makes sense!